pymtl3/passes/backends/verilog/util/test_utility.py

```diff
@@ -290,7 +290,7 @@ def outport_filter( obj ):
   def ref_tv_out( model, test_vector ):
     dct = {}
     for out_port in all_output_ports:
-      dct[ out_port ] = eval( "model." + out_port._dsl.my_name )
+      dct[ out_port ] = eval( "model." + out_port._dsl.my_name ).clone() # WE NEED TO CLONE NOW
```

Comment:
Is this because we always reuse and mutate the same Bits object now?

Reply:
right. I'm adding another function to catch = instead of @= at the top level.